CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ### Features
 
+* `ilab` now uses dedicated storage directories depending on where they are defined for a particular
+   OS given the output of the Python `platformdirs` package. Particularly, `ilab` now has a
+   directory dedicated for config, data, and cache files. The default `ilab` config that gets
+   created during `ilab init` and read in subsequent operations now lives in the dedicated
+   directory which is referred to as `instructlab`. Similarly, a dedicated data directory is
+   used for storing: the models, taxonomy, datasets, and chatlogs by default. The checkpoints
+   are now placed in the system's cache directory as a default.
```

Comment:
I find this hard to understand -- and I'm familiar with platform directories. The changelog should be easy to understand by end users. For starters, the entry should mention XDG and link to your design doc for the change. It should also include examples like `~/.local/share/instructlab` for people who are unfamiliar with XDG.

I suggest that you work with Kelly Brown on the changelog. She is our technical writer and expert for documentation.

Reply:
@tiran We can certainly update it to mention XDG. I intentionally left that detail out since it's platform dependent, so it would be XDG on Linux, Application Support on MacOS, and we don't support Windows yet.